charts/dvpe-netfilter/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.netfilter.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: update-iptables-script
+  namespace: kube-system
+data:
+  update-iptables.sh: |
+    #!/bin/bash
+    apk update
+    apk add iptables
+    {{- range $if := .Values.netfilter.blocked.interfaces }}
+    {{- range $ip := $.Values.netfilter.blocked.ipv4 }}
```

Comment:
remove $ from $.Values.netfilter.blocked.ipv4

Reply:
Hi @mithie ,

the $ is required to work in helm correctly.